piperider_cli/generate_report.py

```diff
@@ -122,3 +122,8 @@ def output_report(target_directory):
             console.print(f"Report generated in {output}/index.html")
         else:
             console.print(f"Report generated in {default_output_directory}/index.html")
+
+        # only open the local file report if auto-upload is OFF
+        if open_report and not auto_upload:
+            result_output = f"{output if output else default_output_directory}/index.html"
```

Comment:
browsers can't find the report when `output` is given a relative path

Reply:
how do we verify that it's a relative path vs. absolute path?

Reply:
https://stackoverflow.com/questions/51520/how-to-get-an-absolute-file-path-in-python